pom.xml

```diff
@@ -101,7 +101,7 @@
         <netty3.version>${netty3-version}</netty3.version><!-- Spark -->
         <minio.version>${minio-version}</minio.version><!-- Keep in sync with quarkiverse-minio and Camel, preferring Camel assuming that it is more likely to be up to date -->
         <mvel2.version>${mvel-version}</mvel2.version>
-        <nimbus-jose-jwt.version>4.41.1</nimbus-jose-jwt.version><!-- Mess in hdfs transitive deps -->
+        <nimbus-jose-jwt.version>9.10</nimbus-jose-jwt.version><!-- It's placed here because of mess in hdfs transitive deps, please upgrade when it's possible -->
```

Comment:
If this is impacting Kafka, it would be nice to mention Kafka in the comment.

If the version 9.10 comes from some Kafka pom, then we should check whether it is possible to use `@sync` to update it automatically.

Reply:
Okay. It's not a kafka client pom, but usage of Strimzi, with oAuth2. The dependency is added for this purpose only : https://quarkus.io/guides/kafka#authenticating-with-oauth
 
That bug will maybe appear too on the Camel-K side, since they're using the Strimzi-oauth-client too.
When upgrading the client from 0.7.x to 0.8.x, the Strimzi oAuth client moves from Keyclock to Nimbus jose. so this configuration fails to start.

```
camel.component.kafka.security-protocol = SASL_SSL
camel.component.kafka.sasl-mechanism = OAUTHBEARER
camel.component.kafka.sasl-jaas-config = org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required \
        oauth.client.id="<YOUR-SERVICE-ACCOUNT-ID-HERE>" \
       oauth.client.secret="<YOUR-SERVICE-ACCOUNT-SECRET-HERE>" \
       oauth.token.endpoint.uri="<TOKEN_ENDPOINT_URI>" ;
camel.component.kafka.additional-properties[sasl.login.callback.handler.class] = io.strimzi.kafka.oauth.client.JaasClientOauthLoginCallbackHandler
```
And when I checked that version we have is way too old. I was wondering if we can't just push this one just for hdfs purpose ?

Reply:
To make sure it works, we need to have an integration tests with testcontainers + Strimzi that has oAuth2, I don't know if it's easy to setup

Reply:
I thought `io.strimzi:kafka-oauth-client` would be managed in Quarkus BOM, but it is not. It it was managed there, we could use our tooling to keep our `nimbus-jose-jwt.version` in sync with theirs https://github.com/strimzi/strimzi-kafka-oauth/blob/main/pom.xml#L90 
I wonder how much sense would it make for quarkus to manage `io.strimzi:kafka-oauth-client`. WDYT?

Reply:
In any case the comment should say something like `mess in hdfs transitive deps, used also by io.strimzi:kafka-oauth-client recommended by https://quarkus.io/guides/kafka#authenticating-with-oauth`